conf/riff-raff.yaml

```diff
@@ -19,13 +18,11 @@ deployments:
   workflow-notification:
     type: aws-lambda
     parameters:
-      bucket: workflow-dist
       prefixStack: false
       functionNames: [workflow-notification-]
       fileName: workflow-notification.zip
   workflow-frontend-fluentbit:
     type: aws-s3
     parameters:
-      bucket: workflow-dist
```

Comment:
I don't think you can remove the bucket from aws-s3 (at least when I tried on grid I got failures when trying to preview the deployment, I can't remember if I actually tried to deploy it)

Reply:
oh that's sad, thanks for spotting - indeed the deploy failed in `CODE` - any plans to add that @akash1810? (I'm guessing the suggestion here would be to use `cdk-base` right?

Reply:
ok reinstated `bucket` for the `aws-s3` deploy step, and it deploys fine. ready for re-review @andrew-nowak - thank you